resources/js/components/SignIn.js

```diff
@@ -0,0 +1,120 @@
+import * as React from 'react';
+import { Link as RouterLink } from 'react-router-dom';
+import Footer from './layouts/Footer';
+
+import Avatar from '@mui/material/Avatar';
+import Button from '@mui/material/Button';
+import CssBaseline from '@mui/material/CssBaseline';
+import TextField from '@mui/material/TextField';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import Checkbox from '@mui/material/Checkbox';
+import Links from '@mui/material/Link';
+import Paper from '@mui/material/Paper';
+import Box from '@mui/material/Box';
+import Grid from '@mui/material/Grid';
+import Typography from '@mui/material/Typography';
+import LockOutlinedIcon from '@mui/icons-material/LockOutlined';
+import { createTheme, ThemeProvider } from '@mui/material/styles';
+
+const theme = createTheme();
+
+export default function SignInSide() {
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    const data = new FormData(event.currentTarget);
+    // eslint-disable-next-line no-console
+    console.log({
+      email: data.get('email'),
+      password: data.get('password'),
+    });
+  };
+
+  return (
+    <ThemeProvider theme={theme}>
+      <Grid container component="main" sx={{ height: '100vh' }}>
+        <CssBaseline />
+        {/* <img alt="imggg" src="images/e-learning_bg.jpeg"></img> */}
+        <Grid
+          item
+          xs={false}
+          sm={4}
+          md={7}
+          sx={{
+            // backgroundImage: 'url(images/e-learning_bg.jpeg)',
```

Comment:
remove if not needed

Reply:
I have remove the backgroundImage commented line